packages/standard-minifier-js/plugin/comibinedFile.js

```diff
@@ -0,0 +1,125 @@
+function countNewLines(code) {
```

Comment:
I don't think you need to have combinedFile. It doesn't support source maps, and without source maps all you need to do is simply concat the code strings, like the core minifier did before.